### ISSUE TITLE
Add chart to System monitor increments the number with each new chart added

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#18561]
### Describe the bug

Each time a new chart is added to System monitor, the number keeps incrementing while it should always be 1.

### To Reproduce

1. Go to https://demo.phpmyadmin.net/QA_5_2/index.php?route=/server/status/monitor
2. Click on `Settings`
3. Click on `Add chart`
4. Click on `Preset chart`
5. Click on `Add chart to grid`
6. Repeat steps 2-5
7. See the chart number incrementing with each additional chart added, while should always be 1.

### Expected behavior

Each time a new chart is added, it should only be 1.

### Screenshots

https://github.com/phpmyadmin/phpmyadmin/assets/25424343/181084e0-4088-4c6f-ad61-8165bd4b837e

### Server configuration

- phpMyAdmin version: 5.2.2-dev, 6.0.0-dev

**Comment by williamdes** [comment]
Thank you for the report of this weird bug
For contributors: please create a new branch from QA_5_2 and submit your pull request to QA_5_2

**Comment by RajSinha67** [comment]
Actually, the issue starts from the moment we click on 'Add Chart' button and the modal to create chart appears. 
When I tried to debug I found that if I open the chart modal (Settings -> Add Chart) and then click on 'Close' button and I do this say 3 times. On the 4th time if I create chart ('Add Chart to grid') it will print 4 new charts.
The number of charts created is depending on : Number of times 'Chart title' modal has appeared before + this time when 'Add chart to grid' button is clicked.
